src/corona_hakab_model/agent.py

```diff
@@ -70,27 +73,29 @@ class AgentSnapshot:
 
 
 class InitialSickAgents:
-    EXPORT_OUTPUT_DIR = "../../output/"
+    EXPORT_OUTPUT_DIR = os.path.join(os.path.split(os.path.dirname(os.path.realpath(__file__)))[0], "output")
```

Comment:
Why not use the constant in project_structure?

Reply:
It wasn't there when I started working on it :laughing: 
where is it?

Reply:
It wasn't there when I started working on it :laughing: 
where is it?

Reply:
You might need to puul from develop? I hope it is in develop and not in dev4.0